src/main/java/mate/academy/repository/BookRepository.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.repository;
+
+import mate.academy.model.Book;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BookRepository extends JpaRepository<Book, Long> {
```

Comment:
revert changes - you should implement BookRepository
```suggestion
public interface BookRepository {
```